docs/installation/mac.md

```diff
@@ -56,10 +57,40 @@ You can test if it's up and running by issuing the following command:
 redis-cli ping
 ```
 
+### Elasticsearch
+
+DEV requires Elasticsearch version 7 or higher.
+
+We recommend installing from archive on Mac. The following directions were
+[taken from the Elasticsearch docs themselves](https://www.elastic.co/guide/en/elasticsearch/reference/7.5/targz.html#install-macos),
+so check those out if you run into any issues or want further information. NOTE:
+Make sure to download the OSS version, `elasticsearch-oss`.
+
+```shell
+wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-7.5.2-darwin-x86_64.tar.gz
+wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-7.5.2-darwin-x86_64.tar.gz
```

Comment:
I think this is missing the `.sha512` from the end of this line according to the [ES docs](https://www.elastic.co/guide/en/elasticsearch/reference/7.5/targz.html#install-macos)

Reply:
GDI, going to fix in a followup since I got the reviews on this and want to get it in there